solr/solrj/src/test/org/apache/solr/client/solrj/impl/HttpSolrClientBuilderTest.java

```diff
@@ -51,6 +52,22 @@ public void testProvidesHttpClientToClient() throws IOException {
     }
   }
 
+  @AwaitsFix(bugUrl = "https://issues.apache.org/jira/browse/SOLR-13605")
+  @Test
+  public void testUsesTimeoutProvidedByHttpClient() throws IOException {
+
+    ModifiableSolrParams clientParams = new ModifiableSolrParams();
+    clientParams.set(HttpClientUtil.PROP_SO_TIMEOUT, 12345);
+    clientParams.set(HttpClientUtil.PROP_CONNECTION_TIMEOUT, 67890);
+    HttpClient httpClient = HttpClientUtil.createClient(clientParams);
+    try (HttpSolrClient createdClient =
+        new Builder(ANY_BASE_SOLR_URL).withHttpClient(httpClient).build()) {
+      assertEquals(createdClient.getHttpClient(), httpClient);
+      assertEquals(67890, createdClient.getConnectionTimeout());
+      assertEquals(12345, createdClient.getSocketTimeout());
+    }
+  }
```

Comment:
missing 
```
HttpClientUtil.close(httpClient);
```

Reply:
i am trying to get it defined in the try/with/resources...

Reply:
interesting. HttpClient does not seem to implement AutoCloseable so you would not be able to do it like this. had no idea

Reply:
YOu are right...    I added the close...   One thing is that I think there are other tests that don't do this closing, like AdminHandlersProxyTest....